### PR TITLE
Add daily random runes

### PR DESCRIPTION
**Pull request by astaninger**

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java [hunk 1]
```diff
@@ -139,6 +139,10 @@ public void onGameTick(GameTick event)
 			sendChatMessage("You have pure essence waiting to be collected from Wizard Cromperty.");
 			hasSentEssenceMsg = true;
 		}
+		if (config.showRunes() && canCollectRunes())
+		{
+			sendChatMessage("You have random runes waiting to be collected from Lundail.");
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java]: You should maybe also add boolean that the message was sent in order to not send it every time

File: runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java [hunk 2]
```diff
@@ -139,6 +142,11 @@ public void onGameTick(GameTick event)
 			sendChatMessage("You have pure essence waiting to be collected from Wizard Cromperty.");
 			hasSentEssenceMsg = true;
 		}
+		if (config.showRunes() && !hasSentRunesMsg && checkCanCollectRunes())
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java]: Add empty line above

File: runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java [hunk 3]
```diff
@@ -59,7 +59,7 @@
 	@Inject
 	private ChatMessageManager chatMessageManager;
 
-	private boolean hasSentHerbMsg, hasSentStavesMsg, hasSentEssenceMsg, check;
+	private boolean hasSentHerbMsg, hasSentStavesMsg, hasSentEssenceMsg, hasSentRunesMsg, check;
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java]: reset hasSentRunesMsg on shutdown

**Comment by raiyni** [comment]
now rebase

**Pull request merged**